### ISSUE TITLE
copyFileSync: could not write to dest file (code=ENOENT):.git/hooks/commit-msg

### ISSUE THREAD

**Issue by bugwelle** [ajoslin/angular-scrolly#13]
Hello,

I know I have been submitting many issues , but I think this might be an issue, too:

Using latest code and running `sudo grunt build` throws this error:

`cp: copyFileSync: could not write to dest file (code=ENOENT):.git/hooks/commit-msg`

By creating the folder `hooks` manually in `.git`, I don't get this error.

Note: I'm using `sudo` because otherwise I get this error:

```
ngminifying src/_main.js,src/directives/scroll.js,src/services/dragger.js,src/services/scroller.js,src/services/transformer.js
Warning: Unable to write "dist/angular-scrolly.js" file (Error code: EACCES). Use --force to continue.
```

I think some files don't have the required permission.

Does someone know how to fix this?

Thanks,
Andre

**Comment by ajoslin** [comment]
What OS are you using, and what version of git?

git should create the hooks folder for you.

**Comment by bugwelle** [comment]
Sorry, I forgot to write that:
- Mac OSX 10.7.5
- Node v0.10.13
- npm v1.3.2
- git version 1.8.3.3

// I'm using the GitHub client on Mac ;)

**Comment by ajoslin** [comment]
Haha I dunno then!  

We should probably just use the grunt.file.copy command, as that creates folders if they don't exist.

Could you open a PR which uses `grunt.file.copy` instead of the shelljs cp?

**Comment by bugwelle** [comment]
Just updated to git 1.8.3.4. Still doesn't work.
Using `grunt.file.copy` is a great idea. I don't have time to open a PR right now, but I think I can do tomorrow :)

**Comment by ajoslin** [comment]
Fixed in b4daccd9913f2bfc40d37efc4b8b47fd5bbe0787
